src/Symfony/Component/Finder/Glob.php

```diff
@@ -60,9 +60,15 @@ public static function toRegex($glob, $strictLeadingDot = true, $strictWildcardS
 
             $firstByte = '/' === $car;
 
-            if ($firstByte && $strictWildcardSlash && isset($glob[$i + 3]) && '**/' === $glob[$i + 1].$glob[$i + 2].$glob[$i + 3]) {
-                $car = $strictLeadingDot ? '/(?:(?=[^\.])[^/]++/)*' : '/(?:[^/]++/)*';
-                $i += 3;
+            if ($firstByte && $strictWildcardSlash) {
+                if (isset($glob[$i + 3]) && '**/' === $glob[$i + 1].$glob[$i + 2].$glob[$i + 3]) {
+                    $car = $strictLeadingDot ? '/(?:(?=[^\.])[^/]++/)*' : '/(?:[^/]++/)*';
+                    $i += 3;
+                } elseif (isset($glob[$i + 2]) && '**' === $glob[$i + 1].$glob[$i + 2]) {
+                    $car = '/.*';
```

Comment:
it already matches inside the line: that's the first case of the "if"
see https://github.com/symfony/symfony/pull/22239/files#r109962164